package.json

```diff
@@ -178,102 +156,96 @@
-    "@nx/js": "16.3.2",
-    "@nx/react": "^16.3.2",
-    "@nx/vite": "16.3.2",
-    "@simbathesailor/babel-plugin-use-what-changed": "^2.1.0",
-    "@testing-library/jest-dom": "^5.16.5",
+    "@commitlint/cli": "^17.6.7",
+    "@commitlint/config-conventional": "^17.6.7",
+    "@lingui/swc-plugin": "^4.0.4",
+    "@lingui/vite-plugin": "^4.3.0",
+    "@nx/cypress": "16.4.0",
+    "@nx/eslint-plugin": "16.4.0",
+    "@nx/jest": "^16.4.0",
+    "@nx/js": "16.4.0",
+    "@nx/linter": "16.4.0",
+    "@nx/react": "16.4.0",
+    "@nx/vite": "16.4.0",
+    "@nx/workspace": "16.4.0",
+    "@swc/cli": "~0.1.62",
+    "@swc/core": "~1.3.51",
     "@testing-library/react": "14.0.0",
     "@testing-library/react-hooks": "^8.0.1",
-    "@testing-library/user-event": "^13.5.0",
     "@typechain/ethers-v5": "^10.2.0",
     "@types/clone-deep": "^4.0.1",
     "@types/d3": "^7.4.0",
     "@types/jest": "^29.4.0",
     "@types/ms": "^0.7.31",
     "@types/ms.macro": "^2.0.0",
     "@types/node": "18.14.2",
-    "@types/react": "18.0.28",
-    "@types/react-dom": "18.0.11",
+    "@types/react": "18.2.14",
+    "@types/react-dom": "18.2.6",
     "@types/react-helmet": "^6.1.6",
+    "@types/react-is": "18.2.1",
     "@types/react-router-hash-link": "^2.4.5",
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@types/react-window": "^1.8.5",
     "@types/rebass": "^4.0.10",
-    "@types/styled-components": "^5.1.26",
+    "@types/styled-components": "5.1.26",
     "@types/ua-parser-js": "^0.7.36",
     "@types/wcag-contrast": "^3.0.0",
-    "@vitejs/plugin-react": "^3.0.0",
-    "@vitest/ui": "^0.31.0",
-    "assert": "^2.0.0",
-    "babel-jest": "^29.4.1",
-    "craco-workbox": "^0.2.0",
-    "create-react-app": "^5.0.1",
-    "crypto-browserify": "^3.12.0",
-    "cypress": "^12.4.1",
+    "@typescript-eslint/eslint-plugin": "^6.2.0",
+    "@typescript-eslint/parser": "^6.2.0",
+    "@vitejs/plugin-react": "~4.0.0",
+    "@vitejs/plugin-react-swc": "^3.3.2",
+    "@vitest/coverage-c8": "~0.32.0",
+    "@vitest/ui": "~0.32.0",
+    "babel-jest": "^29.6.2",
+    "babel-plugin-styled-components": "2.1.4",
+    "babel-plugin-transform-import-meta": "^2.2.0",
+    "cypress": "^12.16.0",
+    "eslint": "~8.15.0",
+    "eslint-config-prettier": "8.1.0",
     "eslint-config-react-app": "^7.0.1",
-    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-cypress": "^2.10.3",
     "eslint-plugin-import": "2.27.5",
     "eslint-plugin-jsx-a11y": "6.7.1",
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "https-browserify": "^1.0.0",
-    "husky": "^8.0.3",
-    "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.5.0",
+    "eslint-plugin-unused-imports": "^3.0.0",
+    "husky": "^8.0.0",
+    "isomorphic-fetch": "^3.0.0",
     "jest-fetch-mock": "^3.0.3",
     "jest-styled-components": "^7.1.1",
-    "jsdom": "~20.0.3",
-    "launchdarkly-js-client-sdk": "^3.1.2",
```

Comment:
Removing launchdarkly client?

Reply:
It still here:
https://github.com/cowprotocol/cowswap/blob/develop/package.json#L255
Actually we use `launchdarkly-react-client-sdk`